test/smoke/src/debugAndroid.test.ts

```diff
@@ -110,5 +110,53 @@ export function setup() {
             client.closeApp();
             client.endAll();
         });
+
+        it("Android pure RN Expo test", async function () {
+            this.timeout(debugExpoTestTime);
+            const app = this.app as SpectronApplication;
+            await app.restart({workspaceOrFolder: pureRNWorkspacePath});
+            await app.workbench.explorer.openExplorerView();
+            await app.workbench.explorer.openFile("App.js");
+            await app.runCommand("cursorTop");
+            console.log("Android pure RN Expo test: App.js file is opened");
+            await app.workbench.debug.setBreakpointOnLine(23);
+            console.log("Android pure RN Expo test: Breakpoint is set on line 23");
+            await app.workbench.debug.openDebugViewlet();
+            console.log(`Android pure RN Expo test: Chosen debug configuration: ${ExpoDebugConfigName}`);
+            await app.workbench.debug.chooseDebugConfiguration(ExpoDebugConfigName);
+            console.log("Android pure RN Expo test: starting debugging");
+            await app.workbench.debug.startDebugging();
+            await app.workbench.waitForTab("Expo QR Code");
+            await app.workbench.waitForActiveTab("Expo QR Code");
+            console.log("Android pure RN Expo test: 'Expo QR Code' tab found");
+            await app.workbench.selectTab("Expo QR Code");
+            console.log("Android pure RN Expo test: 'Expo QR Code' tab selected");
+            let expoURL = await app.workbench.debug.prepareExpoURLToClipboard();
+            assert.notStrictEqual(expoURL, null, "Expo URL pattern is not found in the clipboard");
+            const opts = appiumHelper.prepareAttachOptsForAndroidActivity(EXPO_APP_PACKAGE_NAME, EXPO_APP_ACTIVITY_NAME,
+            smokeTestsConstants.defaultTargetAndroidPlatformVersion, androidEmulatorName);
+            let client = appiumHelper.webdriverAttach(opts);
+            let clientInited = client.init();
+            await appiumHelper.openExpoApplicationAndroid(clientInited, expoURL);
+            console.log(`Android pure RN Expo test: Waiting ${smokeTestsConstants.expoAppBuildAndInstallTimeout}ms until Expo app is ready...`);
+            await sleep(smokeTestsConstants.expoAppBuildAndInstallTimeout);
+            await appiumHelper.enableRemoteDebugJSForRNAndroid(clientInited);
+            await app.workbench.debug.waitForDebuggingToStart();
+            console.log("Android pure RN Expo test: debugging started");
+            await app.workbench.debug.waitForStackFrame(sf => sf.name === "App.js" && sf.lineNumber === 12, "looking for App.js and line 12");
+            console.log("Android pure RN Expo test: stack frame found");
+            await app.workbench.debug.continue();
+            await app.workbench.debug.continue();
```

Comment:
```suggestion
            await app.workbench.debug.continue(); // second continue() is needed because BP is being hit for two time for unknown reason
```